ethcore/src/evm/evm.rs

```diff
@@ -66,6 +67,8 @@ pub enum Error {
 	MutableCallInStaticContext,
 	/// Likely to cause consensus issues.
 	Internal(String),
+	/// Wasm runtime error
+	Wasm(String),
```

Comment:
Is there something better we can give here than `String`? Maybe a `WasmError` enum with some number of "proper" error variants and one `Unknown(String)` variant.